lib/coercion.ex

```diff
@@ -45,7 +45,12 @@ defmodule Coercion do
       {:ok, "true"}
       iex> coerce(10.5, :string)
       {:ok, "10.5"}
-
+      iex(3)> :bellow
```

Comment:
Can you remove the prompt numbers from these examples?

Reply:
Done.